control/control.openSUSE.xml

```diff
@@ -354,6 +353,9 @@ textdomain="control"
                 <!-- the default subvolume "@" was requested by product management -->
                 <btrfs_default_subvolume>@</btrfs_default_subvolume>
 
+                <!-- root filesystem should be read-only -->
```

Comment:
Please, update the comment. In this case, it should not be read-only. And it is wrong in master too :(